emmarin/cl/cl/src/mantle/update.rs

```diff
@@ -0,0 +1,13 @@
+use crate::mantle::ZoneState;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct UpdateBundle {
+    pub updates: Vec<ZoneUpdate>,
+}
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct ZoneUpdate {
```

Comment:
And this is just `Update` in the spec. Maybe we update the spec to be `ZoneUpdate` and `BatchZoneUpdate`.